solr-8/Dockerfile.spatial

```diff
@@ -0,0 +1,60 @@
+FROM solr:8
+
+EXPOSE 8983
+
+# TODO: Update with new release branch when merged
+ARG CKAN_BRANCH="master"
+
+
+ENV SOLR_INSTALL="/opt/solr/"
+ENV SOLR_CONFIG_DIR="$SOLR_INSTALL/server/solr/configsets"
+ENV SOLR_SCHEMA_FILE="$SOLR_CONFIG_DIR/ckan/conf/managed-schema"
+
+ARG JTS_VERSION="1.19.0"
+ARG JTS_JAR_FILE="$SOLR_INSTALL/server/solr-webapp/webapp/WEB-INF/lib/jts-core-$JTS_VERSION.jar"
+
+USER root
+
+# Create a CKAN configset by copying the default one
+RUN cp -R $SOLR_CONFIG_DIR/_default $SOLR_CONFIG_DIR/ckan
+
+# Update the schema
+ADD https://raw.githubusercontent.com/ckan/ckan/$CKAN_BRANCH/ckan/config/solr/schema.xml $SOLR_SCHEMA_FILE
+
+# Install JTS JAR file
+ADD https://repo1.maven.org/maven2/org/locationtech/jts/jts-core/$JTS_VERSION/jts-core-$JTS_VERSION.jar \
+    $JTS_JAR_FILE
+RUN chmod 644 $JTS_JAR_FILE    
+
+# Add the spatial field type definitions and fields
+
+## RPT
+ENV SOLR_RPT_FIELD_DEFINITION '<fieldType name="location_rpt"   class="solr.SpatialRecursivePrefixTreeFieldType" \
+    spatialContextFactory="JTS"     \
+    autoIndex="true"                \
+    validationRule="repairBuffer0"  \
+    distErrPct="0.025"              \
+    maxDistErr="0.001"              \
+    distanceUnits="kilometers" />'
```

Comment:
Compared this with BPA's config in https://github.com/BioplatformsAustralia/dockercompose-bpa-ckan/blob/1dd1f811c269b2be17bacf2072dd55d43ea6cc76/solr/ckan/conf/schema.xml#L84.   This appears based on the instructions for solr-spatial-field in https://github.com/ckan/ckanext-spatial/blob/master/doc/spatial-search.rst

Querying the change of units from degrees to kilometers?